src/Scan.js

```diff
@@ -276,30 +281,74 @@ class Scan extends React.Component {
       });
   }
 
+  async fetchManyRequests(items) {
```

Comment:
I would suggest to name it just fetchRequests

Reply:
There is already the fetchRequests function, it only makes one request. Do I still need to rename fetchManyRequests?

Reply:
Actually now it is not clear why we have fetchRequests and fetchManyRequests and what difference between them. If we really need both of them I would name them something like this `fetchSomethingRequests` and `fetchSomething2Requests` or `fetchRequestsSomething` and `fetchRequestsSomething2` depending on what these functions do. For example `fetchBrokenRequests`, `fetchUserRequests`, `fetchCustomerRequests`, `fetchRequestsByItemId` and so on.